tests/test_scheduler.py

```diff
@@ -0,0 +1,139 @@
+# Copyright 2022 MosaicML LLM Foundry authors
+# SPDX-License-Identifier: Apache-2.0
+
+from functools import partial
+from typing import List
+
+import pytest
+import torch
+from composer.core import State, Time, TimeUnit
+from composer.devices import DeviceCPU, DeviceGPU
+from composer.models import ComposerClassifier
+from composer.optim.scheduler import ComposerScheduler
+
+from llmfoundry.optim.scheduler import InverseSquareRootWithWarmupScheduler
+
+MAX_DURATION = '100ba'
+STEPS_PER_EPOCH = 100
```

Comment:
Prepend private global constants with `_`

Reply:
This is a test file, so I don't think super strict on public/private things is particularly important. That being said, I don't think you actually need a real dataloader.

Reply:
done